### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 10

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#275] [Esercizi - Viaggi/Piazza]
**Esercizio 5.3 del libro di testo (Geometria Analitica: Marco Abate & Chiara De Fabritiis)**

**Comment by daromacarlo** [comment]
![Immagine 2023-12-04 173636](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/559856ea-cd16-4bd5-9c56-3a2d4f556825)

**Reply by CasuFrost** [reply]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/6e913250-9bb8-425d-8164-e9e177889947)
credo che qui non ci debba essere quell'alpha

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/58ffb401-c78c-40c6-a804-95a0cd0967e0)
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/1cd377fb-8402-45b6-b120-7a9a861a8983)
